modules/get-started/pages/whats-new.adoc

```diff
@@ -11,22 +11,33 @@ See also: xref:deploy:deployment-option/cloud/whats-new-cloud.adoc[]
+* Output to multiple topics: Data transforms can now output to multiple topics.  For example, you can distribute different message types to multiple event streams, or you can write valid records to an output topic and invalid records to a dead-letter queue. See the xref:reference:data-transform-golang-sdk.adoc[Golang SDK] and xref:reference:data-transform-rust-sdk.adoc[Rust SDK] for data transforms.
 
-* xref:develop:data-transforms/run-transforms.adoc#view-data-transform-logs[Data transform logs] are now stored in an internally-managed topic and are accessible via `rpk` and the Redpanda Console.
+* xref:develop:data-transforms/run-transforms.adoc#view-data-transform-logs[Data transform logs] are now stored in an internally-managed topic and are accessible via `rpk` and Redpanda Console.
 
 == Topic-aware partition balancing 
 
 Redpanda now prioritizes xref:manage:cluster-maintenance/cluster-balancing.adoc[balancing a topic's partition replica count on each broker] while it's balancing the cluster's overall partition count. Because different topics in a cluster can have vastly different load profiles, this better distributes the workload evenly across brokers. This feature is enabled by default with the `partition_autobalancing_topic_aware` property. After upgrading to 24.1, new topics are distributed in a topic-aware fashion, and the next partition rebalance (triggered by adding new brokers or by calling the `/v1/partitions/rebalance` Admin API endpoint) evens out the partition counts on each broker.
 
+== Managed identities for Azure
+Redpanda now supports managed identities for Azure resources, similar to the existing support for IAM roles for self-hosted deployments on AWS and GCP. This provides a safer alternative to static credentials, like access keys. Managed identities allow Redpanda clusters deployed on Azure virtual machines and on Azure Kubernetes Service (AKS) to access Azure Blob Storage (ABS) and other services without storing credentials in the cluster configuration.
+
+== Topic recovery validation
+
+The properties xref:reference:cluster-properties.adoc#cloud_storage_recovery_topic_validation_mode[`cloud_storage_recovery_topic_validation_mode`] and xref:reference:cluster-properties.adoc#cloud_storage_recovery_topic_validation_depth[`cloud_storage_recovery_topic_validation_depth`] now add a consistency check that runs before a `redpanda.remote.recovery=true` topic is recovered from object storage.
+
 == New commands
 
 The following `rpk` commands are new in this release:
 
+- xref:reference:rpk/rpk-cloud/rpk-cloud-cluster.adoc[`rpk cloud cluster`]
```

Comment:
Might we want to say a sentence up front about these first 2 rpk cloud ones, and generally what they're generally for (we don't refer to them elsewhere).

@towfiqa ? @twmb

Reply:
cloud cluster select is specifically for profile management (it's basically an alias). Not sure it's worth mentioning. If it is, it's worth mentioning in the context of profiles.

Reply:
@mattschumpert should I remove the 2 `rpk cloud` commands from this list for 24.1 updates? (I had just listed all new commands since the last major release.)

Reply:
yep, and ensure they're added to the RPK reference and any section about rpk Profiles

Reply:
I've removed these 2 commands from this 24.1 What's New. These commands are already in the rpk reference, and I've added [new issue for adding use case/examples](https://github.com/redpanda-data/documentation-private/issues/2407).